pkg/chunk/schema_config.go

```diff
@@ -144,6 +145,22 @@ func (cfg *SchemaConfig) ForEachAfter(t model.Time, f func(config *PeriodConfig)
 	}
 }
 
+func validateChunks(cfg PeriodConfig) error {
```

Comment:
I still believe this logic should belong to `PeriodConfig.Validate()`. Currently it does:

```
func (cfg PeriodConfig) validate() error {
	_, err := cfg.CreateSchema()
	return err
}
```

What you could do is:

```
func (cfg PeriodConfig) validate() error {
	if _, err := cfg.CreateSchema(); err != nil {
	    return err
       }

      // Then here you move the extra validation you're currently doing in validateChunks()
}
```

Reply:
@pracucci Thanks for the review. I will move this.